kubemarine/system.py

```diff
@@ -380,12 +380,17 @@ def fetch_firewalld_status(group: NodeGroup) -> NodeGroupResult:
 
 def is_firewalld_disabled(group):
     result = fetch_firewalld_status(group)
-    disabled_status = True
+    disabled_status = False
 
     for node_result in list(result.values()):
-        if node_result.return_code != 4 and "disabled" not in node_result.stdout:
+        if node_result.return_code == 4:
+            disabled_status = True
+        elif node_result.return_code == 3 and "disabled" not in node_result.stdout:
+            return disabled_status, result
```

Comment:
Looks like this does not solve the reported problem. The reporter has code == 3 and not have disabled in stdout.
Meanwhile, I still not understand exact TC https://github.com/Netcracker/KubeMarine/pull/284#issuecomment-1315454292 and it is still not described.